Update Dependencies

laurencetroyv opened the pull request:
Update dependencies patch versions and migrate tailwind version to tailwind v4

elreco commented:
Hello @laurencetroyv thank you so much for this update, doou think you can fix this error please? 
<img width="1214" height="315" alt="image" src="https://github.com/user-attachments/assets/602f95b2-2349-46ff-a087-62eef984a5e7" />

laurencetroyv commented:
Hi @elreco,

I've fixed the errors and updated the necessary files. However, I haven't tested the changes yet, as I'm not sure how to run tests. Could you please review the updates and let me know if everything looks, correct? Also, any guidance on how to properly test it would be appreciated.

Thank you!

emielmolenaar commented:
@elreco Any chances on getting this merged and released? Would be greaty appreciated. Thank you!
